src/gz21_ocean_momentum/testing/main.py

```diff
@@ -84,6 +83,7 @@
 
```

Comment:
Line 75 - is there a reason for the data_location being some randomly named tmp file in /tmp (without any indication for the user where the files went)? It would be good to have the output saved somewhere more obvious, and maybe even tell the user where....

Reply:
The data_location is a temporary file path used for the output during the run of the script. However, at the end, MLFLOW saves the data as an artifact of the run, the location of which is determined by MLFLOW_TRACKING_URI.

Reply:
OK - that does not happen for me, and MLFLOW_TRACKING_URI is also not set in this script (it is in some Jupyter notebooks). If that needs to be done separately it should probably be mentioned in the README. When I run this script as described in the README, the output can only be found in those temporary folders.

Reply:
I added a comment in the README, and instructions how to set this